fs-storage/src/bin/README.md

```diff
@@ -0,0 +1,31 @@
+## Steps to use CLI
+- Create a test.json file of key:values pairs you want to store.
+```json
+{
+  "key1": "value1",
+  "key2": "value2",
+  "key3": "value3"
+}
+```
+
+-  Change Directory
+```bash
+cd fs-storage/src/bin
+```
+
+- Run Write Command
+```bash
+cargo run -- write /tmp/z test.json
+```
+
+- Run Read Command
```

Comment:
I noticed from the code that if no keys were provided for the `read` command, the whole map gets printed to the terminal

Can you add a note about that?